src/main/java/com/ericsson/gerrit/plugins/eiffel/listeners/ChangeMergedEventListener.java

```diff
@@ -40,6 +41,12 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ChangeMergedEventListener.class);
 
+    /**
```

Comment:
Is it possible to have the injhection in the abstract listener?

Reply:
I understand why you would like to have it the parent class but as it is not used there I find it easier to read by having it in the child classes